.github/workflows/release-main.yaml

```diff
@@ -0,0 +1,22 @@
+name: release-main 
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-and-push-main:
+    runs-on: buildjet-4vcpu-ubuntu-2004
```

Comment:
Looking quickly, I couldn't find a great way to test using this particular runner without merging OR subscribing to buildjet using my personal account. I ended up testing against `ubuntu-latest` in my fork.

Reply:
This is a public repo, so I think this should/can be `ubuntu-latest`.

Reply:
+1 to this just being `ubuntu-latest`.